src/main/features/eligibility/store.ts

```diff
@@ -25,11 +24,10 @@ export class CookieEligibilityStore {
       }
       return undefined
     }
-
-    new Cookies(req, res).set(cookieName, JSON.stringify(_.cloneDeepWith(eligibility, excludeDisplayValue)), { sameSite: 'lax', maxAge: cookieTimeToLiveInMinutes })
+    res.cookie(cookieName, _.cloneDeepWith(eligibility, excludeDisplayValue), { httpOnly: true, secure: true, sameSite: 'lax', maxAge: cookieTimeToLiveInMinutes })
   }
 
   clear (req: express.Request, res: express.Response): void {
-    new Cookies(req, res).set(cookieName, '', { sameSite: 'lax' })
+    res.clearCookie(cookieName, { sameSite: 'lax' })
```

Comment:
@robertparkinson, setting empty value via `res.cookie` function does not remove the cookie so I had to use dedicated method.